Commands/Default.sublime-commands

```diff
@@ -4,15 +4,16 @@
         "command": "edit_settings",
         "args": {
           "base_file": "${packages}/LSP/LSP.sublime-settings",
-          "default": "// Settings in here override those in \"LSP/LSP.sublime-settings\",\n\n{\n\t$0\n}\n"}
+          "default": "// Settings in here override those in \"LSP/LSP.sublime-settings\"\n\n{\n\t$0\n}\n"}
```

Comment:
Can you also remove the extra newline?

`\n\n{\n\t$0\n}\n"}` -> `\n{\n\t$0\n}\n"}`